signs-work-order-map/app/page.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
```

Comment:
I'm still fuzzy on when/how to avoid client-only pages, but my vague understanding is that the further you push client-only components down your tree, the more your site is server-side/cacheable/etc. 

we get zero benefit from this in VZE because the entire app uses the client-only Auth0 provider 😞 .

 not saying that's worth it yet (after all, the entire app is a JS webmap), but just want to keep this consideration in mind.

Reply:
I included this because next kept giving me an error saying it was needed, and if you look at an old commit i had the comment "why is next telling me to use this??"

Reply:
yeah, this is tricky. i read a thing about server components once that stuck with me that described server components as the static branches that leads down to the leaves which are client components that need client-side interactivity. 

since app router defaults to rendering components as server components, I think John's suggestion below about moving the Map into its own component file (with the "use client" directive at the top of that file) would allow removing the directive at the top of this file without that error